__tests__/index.spec.js

```diff
@@ -98,26 +78,63 @@ describe('js-to-styles-vars-loader', () => {
         };
 
         it('gets variable data by modulePath with context', () => {
-            const varData = operator.getVarData([{path: './mocks/colors.js' }], context);
+            const varData = operator.getVarData(path.join(context.context, './mocks/colors.js'));
             expect(varData).toEqual({ white: '#fff', black: '#000'});
         });
 
-        it('merges module data if there are more requests', () => {
-            const varData = operator.getVarData([{path:'./mocks/colors.js'}, {path:'./mocks/sizes.js'}], context);
-            expect(varData).toEqual({ white: '#fff', black: '#000', small: '10px', large: '50px'});
-        });
-
-        it('handles methodName if it is given', () => {
-            const varData = operator.getVarData([{ path:'./mocks/corners.js', methodName: 'typeOne'}], context);
+        it('uses value from property', () => {
+            const varData = operator.getVarData(path.join(context.context, './mocks/corners.js'), 'typeOne');
             expect(varData).toEqual({ tiny: '1%', medium: '3%'});
         });
+        it('uses value from nested property', () => {
+            const varData = operator.getVarData(path.join(context.context, './mocks/corners.js'), 'deep.nested');
```

Comment:
I suggest using either using a fake module that uses such a require or test the property getting part of regex (because this case is not covered yet)

Reply:
:+1: